src/Resources/assets/controllers/accordion_controller.js

```diff
@@ -43,38 +29,69 @@ export default class extends Controller {
      */
     toggle(event)
     {
-        /** @type {HTMLElement[]} */
-        const contentTargets = this.contentTargets;
+        const current = event.currentTarget;
+        const header = this.closestTarget(current, 'header');
+        const arrow = this.closestTarget(current, 'arrow');
+        const contents = this.getAccordionContents(header);
 
-        /** @type {HTMLElement[]} */
-        const headerTargets = this.headerTargets;
+        const isOpen = header.getAttribute('aria-expanded') === 'true';
 
-        /** @type {HTMLElement[]} */
-        const arrowTargets = this.arrowTargets;
+        // Open the current accordion if it wasn't open before (else we toggle it)
+        if (!isOpen) {
+            this.applyOpenState({arrow, contents, header});
+        } else {
+            this.applyCloseState({arrow, contents, header});
+        }
+    }
+
+    /**
+     * @param {HTMLElement[]} elements
+     */
+    applyOpenState(elements) {
+        const {arrow, contents, header} = elements;
 
         /** @type {string[]} */
         const arrowRotateClasses = this.arrowRotateClasses;
 
         /** @type {string[]} */
         const contentHiddenClasses = this.contentHiddenClasses;
 
-        const current = event.currentTarget;
-        const header = this.closestTarget(current, 'header');
-        const arrow = this.closestTarget(current, 'arrow');
-        const content = header.nextElementSibling;
+        header && header.setAttribute('aria-expanded', 'true');
```

Comment:
Use proper if-condition. This is JSX style code and not something we should have in raw JavaScript.